Mid_1.ipynb

```diff
@@ -0,0 +1,1891 @@
+{
```

Comment:
<p>10 points, reduced 10 for not plotting.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rohithandanala/textBasedAnalysis/pull/18/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>